### ISSUE TITLE
Extent Report Version 4 -->Filter by category is not working in Dashboard

### ISSUE THREAD

**Issue by chandrasekhar11** [extent-framework/extentreports-java#138]
Here Iim using GherkhinKeywords for creatng tests , to get BDD Style Report  

extentTest = extentReport.createTest(new GherkinKeyword("Feature"),FeatureName));
extentTest.assignCategory(category);

Dashbord report screenshot with out filter.
![image](https://user-images.githubusercontent.com/61120993/78859122-0b112e80-7a4c-11ea-900b-4bc3e311f05f.png)

When filtered through category in Dashboard , the screenshot is  in Dashboard
![image](https://user-images.githubusercontent.com/61120993/78859193-46abf880-7a4c-11ea-8efa-5419825b73a1.png)

in second tab of the report , the fitering showing correctly
![image](https://user-images.githubusercontent.com/61120993/78859257-6c390200-7a4c-11ea-800b-13ce81b2b85c.png)

**Comment by anshooarora** [comment]
@chandrasekhar11 ExtentHtmlReporter has been deprecated, see [changelog](https://github.com/extent-framework/extentreports-java/blob/master/CHANGELOG.md#413).  Please use ExtentSparkReporter, it builds the same SPA-style view by default.  Please let me know if you are still facing this issue.

**Comment by chandrasekhar11** [comment]
Yes, the issue is there in ExtentSparkReporter as well, filters are not working. Please find the below screenshots.

1. Without Filters
![image](https://user-images.githubusercontent.com/61120993/78973679-02912480-7b2e-11ea-81bf-bc028b3136e8.png)

2. After Applying Filter
![image](https://user-images.githubusercontent.com/61120993/78973745-22284d00-7b2e-11ea-8588-1ec92643406d.png)

**Comment by anshooarora** [comment]
Does the node have this tab, or parent?

**Comment by bijitp** [comment]
So how was it resolved? I am facing the same issue
